app/helpers/orchid/application_helper.rb

```diff
@@ -10,4 +10,29 @@ def clear_search_text
     return new_params
   end
 
+  def site_section
+    if @site_section.present?
+      # Use override from instance variable
+      section = @site_section
+    else
+      if current_page?(home_path)
```

Comment:
since this is a rails method, would it fit the style guide better without parentheses?  I prefer to read it this way, honestly, but I wanted to see if I have the wrong idea in mind for the style guide's preferences.

Reply:
You're right. I think I wrote it that way because I looked at the API and all their examples are written with parentheses. http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-current_page-3F

I'll fix it :)

Reply:
If you decide you would be okay with switching everything ever to parentheses, let me know, and I will start search and replacing immediately with gusto!